ui/v2.5/src/components/Performers/PerformerDetails/PerformerEditPanel.tsx

```diff
@@ -62,7 +63,7 @@ export const PerformerEditPanel: React.FC<IPerformerDetails> = ({
   // Editing state
   const [scraper, setScraper] = useState<GQL.Scraper | undefined>();
   const [newTags, setNewTags] = useState<GQL.ScrapedSceneTag[]>();
-
+  const [rating, setRating] = useState<number>(performer?.rating ?? NaN);
```

Comment:
Should be using formik values instead.

This use of this means that if you navigate away from the edit page after changing the rating but not saving, then no prompt is displayed about unsaved data.